packages/nextra/src/server/schemas.ts

```diff
@@ -106,7 +106,8 @@ export const nextraConfigSchema = z
       recmaPlugins: z.custom<ProcessorOptions['recmaPlugins']>(),
       format: z.enum(['detect', 'mdx', 'md']),
       rehypePrettyCodeOptions: z.custom<RehypePrettyCodeOptions>()
-    })
+    }),
+    autoImportThemeStyle: z.boolean()
```

Comment:
Maybe we should use Zod's `.default(true)`? And for other properties declared in `DEFAULT_CONFIG` object, so this object can be removed?

Reply:
It seems that we need to remove .deepPartial() for the expected default values to work. This might lead to many unintended changes beyond this PR, and I'm worried about potentially breaking something 😰

Reply:
you are right, I did this refactor in https://github.com/shuding/nextra/pull/2569 PR

Reply:
The `nextraConfigSchema` in #2569 looks way better!